service_discovery/proto/build.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    tonic_build::compile_protos("../proto/chariott/v1/chariott_registry.proto")?;
+    tonic_build::compile_protos("../proto/samples/v1/hello_world_service.proto")?;
```

Comment:
what is the link between this samples folder and the one under the service_discovery folder? can we have all sample under that folder?

Reply:
moved